### PR TITLE
#173 split screen

### PR DESCRIPTION
**Pull request by PhillipsOwen**
addresses issue #173 .

this branch has been tested, is stable and can be advertised as "experimental" in order to draw out answers to questions as listed in issue #196 .

### CODE REVIEW

**Comment by PhillipsOwen** [comment]
for some reason my implementation of getting the SLD config for each layer stopped working. 

i converted the layer loading to use useState(). that resulting in the map not resetting properly.

**Review by mbwatson — COMMENTED** [review]
first, it works great. i haven't gotten into the code too much, as i've been poking around the UI trying to test functionality, and i haven't been able to break it. however, i noticed that the model selection dropdowns over in the other tray don't seem to function on this branch.

**Comment by PhillipsOwen** [comment]
> first, it works great. i haven't gotten into the code too much, as i've been poking around the UI trying to test functionality, and i haven't been able to break it. however, i noticed that the model selection dropdowns over in the other tray don't seem to function on this branch.

i had tweaked the zindex this morning thinking it would be straight-forward but it wasnt.

i un-did that update and will be addressed in another PR. please pull this branch again.

File: src/components/trays/compare-layers/CompareLayersTray.js [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+        setRightLayerProps(origLeftLayerProps);
+    };
+
+    /**
+     * get the layer icon
+     *
+     * @param productType
+     * @returns {JSX.Element}
+     */
+    const getLayerIcon = ( productType )=> {
```
Comment (mbwatson) [inline src/components/trays/compare-layers/CompareLayersTray.js:166]: we do something like this quite a bit in this app. i'm thinking this function could probably live in uselayers for everyone to use. it wouldn't have to be done in this PR necessarily, of course, but a nice function to have around.

**Pull request merged**